### DISCUSSION TITLE
我该如何把vscode的copliot改回旧版的侧边活动栏内聊天 而不是自己单开一个窗口

### DISCUSSION THREAD

**Discussion by oldfishoh** [community/community#143444] [Copilot Conversations]
### Select Topic Area

Question

### Body

我该如何把vscode的copliot改回旧版的侧边活动栏内聊天 而不是自己单开一个窗口

**Comment by Akash1134** [comment]
Hi @oldfishoh , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
